examples/albert/run_training_monitor.py

```diff
@@ -99,7 +99,8 @@ def __init__(
         self.state_averager = TrainingStateAverager(
             dht=dht,
             optimizer=opt,
-            prefix=experiment_prefix,
+            scheduler=get_linear_schedule_with_warmup(opt, num_warmup_steps=5000, num_training_steps=125_000),
```

Comment:
It's better to replace these values with the defaults from the config to avoid possible divergence in the future and general "magic constant" readability issues. The best way might be to move the number of training and warmup steps to a common config used both in the monitor and the trainer code

Reply:
I think we shouldn't just move args to a common config, but move the whole code creating `hivemind.Optimizer` to a common file. Otherwise, it's likely that any model/optimizer/scheduler changes made by a user will lead to a divergence between monitors and trainers

Reply:
This dataclass is currently not available to aux peer. For long-term optimal composition, please see CALM and/or petabert examples